netbox/extras/views.py

```diff
@@ -1143,19 +1145,71 @@ def get(self, request, module, name, path=''):
         return redirect(f'{url}{path}')
 
 
-class ScriptResultView(generic.ObjectView):
+class ScriptResultView(TableMixin, generic.ObjectView):
     queryset = Job.objects.all()
 
     def get_required_permission(self):
         return 'extras.view_script'
 
+    def get_table(self, job, request, bulk_actions=True):
+        data = []
+        tests = None
+        table = None
+        index = 0
+        if job.data:
+            if 'log' in job.data:
+                if 'tests' in job.data:
+                    tests = job.data['tests']
+
+                for log in job.data['log']:
+                    index += 1
+                    result = {
+                        'index': index,
+                        'time': log.get('time', None),
+                        'status': log.get('status', None),
+                        'message': log.get('message', None),
```

Comment:
Specifying `None` as the default is redundant if `log` is a dict.
```suggestion
                        'time': log.get('time'),
                        'status': log.get('status'),
                        'message': log.get('message'),
```